af/fastpath/database_upgrade_schema.py

```diff
@@ -246,6 +246,12 @@ def db_setup():
 GRANT SELECT ON TABLE public.counters TO readonly;
 GRANT SELECT ON TABLE public.counters TO amsapi;
 
+GRANT SELECT ON TABLE public.counters_asn_noinput TO readonly;
+GRANT SELECT ON TABLE public.counters_asn_noinput TO amsapi;
+
+GRANT SELECT ON TABLE public.counters_noinput TO readonly;
+GRANT SELECT ON TABLE public.counters_noinput TO amsapi;
+
 GRANT SELECT ON TABLE public.fastpath TO readonly;
 GRANT SELECT ON TABLE public.fastpath TO amsapi;
 GRANT SELECT ON TABLE public.fastpath TO "oomsm-beta";
```

Comment:
Can you add some docs on how to run this? 

I tried doing:
```
python database_upgrade_schema.py db_setup
usage: database_upgrade_schema.py [-h] [--list]
database_upgrade_schema.py: error: unrecognized arguments: db_setup
```
but get that error